lib/cisco_node_utils/interface.rb

```diff
@@ -205,6 +206,32 @@ def fabricpath_feature_set(fabricpath_set)
       FabricpathGlobal.fabricpath_feature_set(fabricpath_set)
     end
 
+    def fabric_forwarding_anycast_gateway
+      config_get('interface', 'fabric_forwarding_anycast_gateway', @name)
+    end
+
+    def fabric_forwarding_anycast_gateway=(state)
+      begin
+        Feature.fabric_forwarding_enable
+        no_cmd = (state ? '' : 'no')
+        config_set('interface',
+                   'fabric_forwarding_anycast_gateway', @name, no_cmd)
+        expected_state = state
+        fail if fabric_forwarding_anycast_gateway.to_s != expected_state.to_s
+      end
+    rescue Cisco::CliError => e
+      vxlan_global = VxlanGlobal.new
+      if vxlan_global.anycast_gateway_mac == ''
```

Comment:
With the change in #220 this will be:

```ruby
rescue Cisco::CliError => e
  anycast_gateway_mac = OverlayGlobal.new.anycast_gateway_mac
  if anycast_gateway_mac.nil? || # nv overlay evpn is disabled
     anycast_gateway_mac.empty? # nv overlay evpn is enabled but mac not set
    raise ...
```